lib/views/general/_frontpage_intro_sentence.html.erb

```diff
@@ -1,4 +1,3 @@
-<h2>
-  Your <strong>Right to Know</strong>
-</h2>
-<p>Every citizen has the right to access information held by public bodies. <strong>By law, they have to respond</strong>. <a href="<%= help_about_url %>">Find out more about Freedom of Information.</a></p>
+<h1 class="intro__title"><%= _("Your right to know") %></h1>
+<p class="intro__description"><%= _("Every citizen has the right to access information held by public bodies. By law, they have to respond") %></p>
+
```

Comment:
I don't think alavetelitheme should override this template, as this isn't adding anything special. We should use the core version. I actually prefer this copy, so happy to port it to core.